src/actions/index.js

```diff
@@ -51,6 +54,11 @@ export const downloadFileOffline = () => ({ type: DOWNLOAD_FILE_E_OFFLINE, alert
 
 export const openFolder = (folderId = ROOT_DIR_ID, isInitialFetch = false, router = null) => {
   return async dispatch => {
+    let routePrefix = '/files'
```

Comment:
`openFolder` is now used to navigate in `/files`and `/trash`, so having a hard-coded route prefix wasn't an option any more. The prefix could be supplied as an argument, but since we already have the router at hand, I think it's safe to extract the prefix from there.